### ISSUE TITLE
Tests fail on Mac with default setup

### ISSUE THREAD

**Issue by Angus-fletcher** [cjdb/project_template#1]
Running Mac Sierra
Just installed it per instructions using g++-4.9 (Homebrew GCC 4.9.4)
made build/debug then ran
`make && make test`
pastebin of error log: https://pastebin.com/2Dvi0dFn
Also when compiling get a whole bunch of errors like this
`/var/folders/k2/8stwhnf16hl5srvm31cdkmqw0000gn/T//cc7iHLxz.s:7278:11: note: change section name to "__data"
        .section __DATA,__datacoal_nt,coalesced
                 ^      ~~~~~~~~~~~~~`
But google says they are harmless and unavoidable

**Comment by cjdb** [comment]
Can you please run this from the `basic_project` directory, and let me know the results?

```
build/debug/test/test.employee < source/australian_scientists.txt > output
```

**Comment by Angus-fletcher** [comment]
Nothing happens returns 134

**Comment by Angus-fletcher** [comment]
issue resolved make sure you are using gcc version 4.9.2 (like outlike says) I was using 4.9.4
Download the correct version https://sourceforge.net/projects/hpc/files/hpc/gcc/ (just pick the one marked gcc-4.9-bin) 
Then:
`gunzip gcc-4.9-bin.tar.gz`
then run 
`sudo tar xvf gcc-4.9-bin.tar -C /`
you can now check its right with `g++ -v`
